test/Scoop-Config.Tests.ps1

```diff
@@ -1,20 +1,45 @@
 . "$psscriptroot\..\lib\core.ps1"
-. "$psscriptroot\..\lib\config.ps1"
 
-describe "hashtable" -Tag 'Scoop' {
-    $json = '{ "one": 1, "two": [ { "a": "a" }, "b", 2 ], "three": { "four": 4 }, "five": true, "six": false, "seven": "\/Date(1529917395805)\/" }'
+describe "config" -Tag 'Scoop' {
+    BeforeAll {
+        $json = '{ "one": 1, "two": [ { "a": "a" }, "b", 2 ], "three": { "four": 4 }, "five": true, "six": false, "seven": "\/Date(1529917395805)\/", "eight": "2019-03-18T15:22:09.3930000+01:00" }'
+    }
+
+    it "converts JSON to PSObject" {
+        $obj = ConvertFrom-Json $json
+
+        $obj.one | Should -BeExactly 1
+        $obj.two[0].a | Should -Be "a"
+        $obj.two[1] | Should -Be "b"
+        $obj.two[2] | Should -BeExactly 2
+        $obj.three.four | Should -BeExactly 4
+        $obj.five | Should -BeTrue
+        $obj.six | Should -BeFalse
+        [System.DateTime]::Equals($obj.seven, $(New-Object System.DateTime (2018, 06, 25, 09, 03, 15, 805))) | Should -BeTrue
+        [System.DateTime]::Equals([System.DateTime]::parse($obj.eight), $(New-Object System.DateTime (2019, 03, 18, 15, 22, 09, 393))) | Should -BeTrue
```

Comment:
the test failed.

Reply:
This is due to TimeZone.

```powershell
$(New-Object System.DateTimeOffset (2019, 03, 18, 15, 22, 09, 393, $(New-Object TimeSpan (1, 0, 0)))).LocalDateTime
```

Reply:
Now it will only test if it's a DateTime object. Not gonna bother about Timezones ...